packages/andive/src/components/icon.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { ReactNodeLike } from 'prop-types'
+import styled from 'styled-components'
+
+import Box from './box'
+
+const Icon = styled.div<Pick<Props, 'align' | 'direction'>>`
```

Comment:
align is a valid HTML attribute, you must filter it like-so:

`const Icon = styled(({ align, direction, ...props }) => <div {...props} />)'   '`